ts/models/conversation.ts

```diff
@@ -1437,6 +1438,10 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
 
     return typeof expireTimer === 'number' && expireTimer > 0;
   }
+
+  private getIsPinned() {
+    return window.inboxStore?.getState().conversations.conversationLookup[this.id].isPinned;
+  }
```

Comment:
better to add an attribute to the conversation object itself. We don't want to split some convo attributes in redux and some in the db, and they are better in the db for now